src/systems/thruster/Thruster.hh

```diff
@@ -112,15 +123,17 @@ namespace systems
   /// gz sim auv_controls.sdf
   /// ```
   /// To control the rudder of the craft run the following:
-  /// ```
-  /// gz topic -t /model/tethys/joint/vertical_fins_joint/0/cmd_pos
-  ///    -m gz.msgs.Double -p 'data: -0.17'
-  /// ```
+  /** ```
```

Comment:
Is it necessary to mix comment styles?

Reply:
There were 2 gcc warnings about multi-line comment, when in fact it's not a multi-line comment, it's just a multi-line code block. gcc doesn't like the `\` character at the end of the line in the code block. It's two lines of code, and if we don't add the `\`, when people copy-paste into the terminal, it probably will give an error. If we put them into one line, then it'll be over 80 characters.

So I tried to make it a multi-line comment to see if those warnings go away. But looks like the build didn't produce outputs... maybe I'll rerun it.

Reply:
These were the warnings https://build.osrfoundation.org/job/gz_sim-ci-pr_any-jammy-amd64/309/gcc/

Reply:
This comment style has gotten rid of the warnings https://build.osrfoundation.org/job/gz_sim-ci-pr_any-jammy-amd64/315/

There are old warnings about buoyancy_engine from #2183. I'll get rid of those warnings the same way too. I guess that's why they didn't add the `\` before.
Sigh, it's a tradeoff between the code lines being copy-pastable in documentation, versus the code looking clean. I think correctness in documentation should take precedence because probably more people look at that.

Reply:
Okay, I agree that correctness in documentation is more important, so let's keep the comment style that makes it work. Maybe someday we can convince cppcheck that long comments are okay, and put the command on one line.

Reply:
67c84499c